docs/api-reference.md

```diff
@@ -608,4 +624,140 @@ end
 As with `setState`, you can set use the constant `Roact.None` to remove a field from the state.
 
 !!! note
-	`getDerivedStateFromProps` is a *static* lifecycle method. It does not have access to `self`, and must be a pure function.
\ No newline at end of file
+	`getDerivedStateFromProps` is a *static* lifecycle method. It does not have access to `self`, and must be a pure function.
+
+---
+
+## ShallowWrapper
+
+### Fields
+
+#### type
+```
+type: {
+	kind: ElementKind
+}
+```
+
+The type dictionary always has the `kind` field that tell the component type. Additionally, depending of the kind of component, other information can be included.
+
+| kind | fields | description |
+| --- | --- | --- |
+| Host | className: string | the ClassName of the instance |
+| Function | functionComponent: function | the function that renders the element |
+| Stateful | component: table | the class-like table used to render the element |
+
+---
+
+#### props
+The props of the ShallowWrapper.
+
+!!! note
+	This dictionary will not contain the `Roact.Children` prop. To obtain the children elements wrapped into a ShallowWrapper, use the method `getChildren()`
```

Comment:
Would it be simpler to just populate children as a property of ShallowWrapper when we call new? I'm trying to think of when you want a ShallowWrapper but wouldn't call getChildren()

Reply:
Should I just make a `children` field and remove `getChildren` and `getChildrenCount`?